aptos-move/framework/aptos-framework/sources/configs/version.move

```diff
@@ -45,6 +49,23 @@ module aptos_framework::version {
         reconfiguration::reconfigure();
     }
 
+    /// Updates the major version to a larger version.
+    /// This can be called by on chain governance.
+    public entry fun set_for_next_epoch(account: &signer, major: u64) acquires Version {
```

Comment:
`set_version_with_dkg`?

Reply:
this is very specific to the buffering sub-feature, i think we don't need to mention it in the function name? Maybe mention it in the function doc.

Reply:
that said, i wonder if `set_version_for_next_epoch` actually makes sense: what if current version is X, one `set_version_for_next_epoch` to X+100, and when DKG finishes the version is already X+200?

@movekevin when do we `set_version`?